web/templates/modules/index.html

```diff
@@ -100,6 +100,72 @@ <h3 class="panel-title">Error</h3>
     </div>
 
 
+    <div class="row">
+        <div class="col-md-12">
+            <div class="card">
+                <div class="header">
+                    <h4 class="title">Preloading Modules</h4>
+                    <p class="category"></p>
+                </div>
+                <div class="content module-list">
+                    {% for module in data.modules['Preloading'] %}
+                        <div class="module-info" id="{{module.name}}">
+                            <div class="row">
+                                <div class="col-sm-8">
+                                    <h5>{{module.name}}</h5>
+                                </div>
+                                <div class="col-sm-4 text-right module-actions">
+                                    <a class="btn btn-xs btn-fill" href="{{ url_for('ModulesView:configure', id=module._id)}}"><i class="fa fa-sliders"></i> Configure</a>
+                                    {% if module.enabled %}
+                                        <form method="post" action="{{ url_for('ModulesView:disable', id=module._id) }}">
+                                            <button type="submit" class="btn btn-danger btn-xs btn-fill"><i class="fa fa-remove"></i> Disable</button>
+                                        </form>
+                                    {% else %}
+                                        {% if not module.error %}
+                                            <form method="post" action="{{ url_for('ModulesView:enable', id=module._id) }}">
+                                                <button type="submit" class="btn btn-success btn-xs btn-fill"><i class="fa fa-check"></i> Enable</button>
+                                            </form>
+                                        {% endif %}
+                                    {% endif %}
+                                </div>
+                            </div>
+                            {% if module.description %}
+                                <div>{{module.description}}</div>
+                            {% endif %}
+
+                            {% if module.error %}
+                            <div class="panel panel-danger">
+                                <div class="panel-heading">
+                                    <h3 class="panel-title">Error</h3>
+                                </div>
+                                <div class="panel-body">
+                                    <pre>{{module.error}}</pre>
+                                </div>
+                            </div>
+                            {% endif %}
+
+                            <ul>
+                                {% if module.acts_on %}
+                                    <li><span class="label">Acts On</span> {{module.acts_on|smart_join}}</li>
+                                {% endif %}
```

Comment:
I don't think Preloading Modules have an acts_on

Reply:
Slipped through while simplifying the preloading module implementation. Will remove it.